mariadb-operator.yaml

```diff
@@ -0,0 +1,165 @@
+package:
+  name: mariadb-operator
+  version: "0.38.1"
+  epoch: 0
+  description: A Go operator that automates the deployment and management of MariaDB databases on Kubernetes
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/mariadb-operator/mariadb-operator.git
+      tag: v${{package.version}}
```

Comment:
the tags there are mixed; there's also one `mariadb-operator-vx.y.z` please validate if this is the actual one.